RFC-000-rfc-process-suggestions.md

```diff
@@ -0,0 +1,77 @@
+# RFC Zero - Open for suggestions to the RFC process
+
+## Proposed by
+
+[Alyssa Rock](https://thegooddocs.slack.com/team/U012KCMPP0V)
+
+## Current status

```

Comment:
I'm a 💯 fan of the checklist, torn on where I like it.  I think it is perfect in the pull-request description so that it can be easily _checked/actioned_ by folks(github-members).  And we keep track of the status as a field inside of the YAML when applicable.  Open to discuss.

Reply:
I agree that it should be either a field inside YAML or included in the description for the pull request.